records_mover/log_levels.py

```diff
@@ -24,7 +23,3 @@ def set_levels(app_name: str,
     # Log all messages to stderr
     root_logger = logging.getLogger()
     root_logger.setLevel(adjusted_log_level(default_root_log_level, app_name))
-
-    # Log app messages to job_log by default
-    app_logger = logging.getLogger(app_name)
-    app_logger.setLevel(adjusted_log_level(default_app_log_level, app_name))
```

Comment:
This was a leftover from BlueHQ job logging - we could set the log level of things that went to job_context.logger separately, and that was the logger which would actually go to the web console..